qiling/os/windows/x86.py

```diff
@@ -175,3 +175,6 @@ def runner(ql):
 
     if ql.internal_exception is not None:
         raise ql.internal_exception
+
+    ql.dprint("Syscalls and number of invocations")
```

Comment:
ql.print with prefix for the msg, [+] and the list with [-] so we know its a subset

Reply:
maybe drpint move before the exception if else